mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraAttachmentMapper.java

```diff
@@ -111,7 +109,7 @@ private Attachment attachment(Row row) {
 
         return FluentFutureStream
             .of(attachments)
-            .flatMap(OptionalConverter::toStream)
+            .flatMap(OptionalUtils::toStream)
```

Comment:
then we don't need neither to keep OptionalConverter::toStream? Maybe we can get rid of all the OptionalConverter class?

Reply:
Basically yes, we can but later on we have to detail on using like
```
.flatMap(OptionalUtils::toStream)
```
should be

```
.flatMap(op -> op.map(Stream::of).orElse(Stream.of()))
```

It makes to code more complicated, so I keeped the old one

Reply:
I like the way it was renamed OptionalUtils.